public/schemas/capital.json.erb

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "file://#{@schema_dir}/capital.json",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Legal Aid Check Financial Eligibility add capital details to an assessment schema",
+  "type": "object",
+  "properties": {
+    "bank_accounts": {
+      "$ref": "<%= "file://#{@schema_dir}/common.json#bank_accounts" %>"
```

Comment:
Are bank accounts or non_liquid_capital used in any other cfe endpoints? If so, I would define them here in capital.json.erb rather than in common.json (see https://github.com/ministryofjustice/check-financial-eligibility/blob/e3c395df416d07a939470970c111d2a0ff34a214/public/schemas/applicant.json.erb)